src/AllLayers.js

```diff
@@ -129,7 +178,8 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
         ) : L.control.embed().addTo(map);
       }
 
-      L.control.layers(baseMaps, this.overlayMaps).addTo(map);
```

Comment:
Let's just add if option has param to show default, use this line else use the below layerbrowser line.

Reply:
Working on this.